flask_jwt_extended/view_decorators.py

```diff
@@ -32,12 +32,13 @@ def jwt_required(fn):
     """
     @wraps(fn)
     def wrapper(*args, **kwargs):
-        jwt_data = _decode_jwt_from_request(request_type='access')
-        ctx_stack.top.jwt = jwt_data
-        if not verify_token_claims(jwt_data[config.user_claims_key]):
-            raise UserClaimsVerificationError('User claims verification failed')
-        _load_user(jwt_data[config.identity_claim_key])
-        return fn(*args, **kwargs)
+        if request.method not in config.exempt_methods:
+            jwt_data = _decode_jwt_from_request(request_type='access')
+            ctx_stack.top.jwt = jwt_data
+            if not verify_token_claims(jwt_data[config.user_claims_key]):
+                raise UserClaimsVerificationError('User claims verification failed')
+            _load_user(jwt_data[config.identity_claim_key])
+            return fn(*args, **kwargs)
```

Comment:
If we don't return `fn(*args, **kwargs)` in an `else` block here, this will break if someone is creating their own `OPTIONS` endpoint via someting like `@app.route('/foo', methods=['GET', 'OPTIONS'])`. I doubt anyone is doing that, but because flask allows this to happen, I would rather get that fixed up anyways, just in case :)

Reply:
You're right! I didn't mean to indent also `return fn(*args, **kwargs)`!